torch/csrc/jit/passes/quantization.cpp

```diff
@@ -466,6 +466,12 @@ class InsertObserversHelper {
                   bool is_entry_point = false,
                   std::unordered_set<Value*> graph_observed_values = std::unordered_set<Value*>());
 
+  std::tuple<OptionalModuleVector, OptionalModuleVector, std::vector<size_t>> insertObserversFor(
```

Comment:
Please add a test with if statements to verify the functionality here

Reply:
support for prim::If will come later

Reply:
this is just a refactor, not adding new functionalities.